### ISSUE TITLE
Babel plugin warnings emitted for local builds

### ISSUE THREAD

**Issue by rwjblue** [emberjs/data#6080]
Steps to reproduce:

```
# clone the repo
yarn
yarn build
```

The following console warnings are emitted:

```
WARNING: ember-data has added the class-properties plugin to its build, but ember-cli-babel provides these by default now! You can remove the transforms, or the addon that provided them, such as @ember-decorators/babel-transforms.
WARNING: ember-data has added the class-properties plugin to its build, but ember-cli-babel provides these by default now! You can remove the transforms, or the addon that provided them, such as @ember-decorators/babel-transforms.
```

ember-cli-babel emits the warning because ember-cli-typescript adds the `class-properties` plugin. ember-cli-typescript has an open PR fixing the issue that is pending https://github.com/typed-ember/ember-cli-typescript/pull/640.

---

Note: this is only an issue when locally developing on ember-data, **not** when it is consumed in an actual app.

**Comment by runspired** [comment]
`ember-cli-typescript` merged a fix ~3.5 months ago but has not yet released it. cc @chriskrycho

**Comment by chriskrycho** [comment]
Thanks for the ping. I plan to cut a release next week (two full days allocated to TS next week)!
